Imagetospeech/image_to_text.py

```diff
@@ -0,0 +1,15 @@
+import pytesseract
+from PIL import Image
+
+
+pytesseract.pytesseract.tesseract_cmd = (r'Enter the path: ')
+
+img = input(r"Enter image: ")
+target = Image.open(img)
+text = pytesseract.image_to_string(target, config='')
+
+with open(f"./Imagetospeech/Text/text.txt",'w') as f:
```

Comment:
Initially, there is no folder named Text, so that script will throw error

Reply:
@kaustubhgupta there is a folder named Text